addons/grenades/functions/fnc_changeFuse.sqf

```diff
@@ -0,0 +1,67 @@
+#include "script_component.hpp"
+/*
+ * Author: Cyruz
+ * Allows conversion of explosive charges in to throwable versions
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * call ace_grenades_fnc_changeFuse
+ *
+ * Public: No
+ */
+
+{
+    _x params ["_mag", "_throwableMag"];
+
+    [
+        _mag,
+        "CONTAINER",
+        "Covert to short fuse",
+        nil,
+        QPATHTOEF(explosives,UI\Place_Explosive_ca.paa),
+        [
+            {true},
+            {
+                params ["", "", "_item", "", "_magArr"];
+                _item isEqualTo (_magArr #0);
```

Comment:
```suggestion
                _item isEqualTo (_magArr select 0);
```
We don't use `#` in ACE.

Reply:
Huh?

Reply:
> Huh?

We do? I've done a search and I could find it only in one file in the whole codebase: https://github.com/acemod/ACE3/blob/master/addons/repair/functions/fnc_addRepairActions.sqf#L46

Maybe not "we don't use but I would say `select` is proffered.

Reply:
Switched to `select`

Reply:
@veteran29 You didn't find anything because you don't need the space, it also doesn't have to be a number. And no, select is not preferred. It's just an older command, # didn't exist when the majority of the code base was written. This is the guideline you should follow and nothing else: https://ace3mod.com/wiki/development/coding-guidelines.html